### ISSUE TITLE
(mobile) IMMICH headline font not updated

### ISSUE THREAD

**Issue by aminomaboul** [immich-app/immich#8206]
### The bug

the immich text headline was not updated as same as #7919 on the top of the profile window

### The OS that Immich Server is running on

ubuntu 20.04

### Version of Immich Server

v1.99.0

### Version of Immich Mobile App

v1.99.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1.open mobile app
2.on the app bar top right, click on the user profile circle
3.the window appear, and the title of this window still having the old font
```

**Comment by Explosion-Scratch** [comment]
![Screenshot_20240324_141734](https://github.com/immich-app/immich/assets/61319150/7713ecd2-9ace-4ad9-be3e-95dad644dad3)
Here is a screenshot for reference

**Comment by aminomaboul** [comment]
fixed by PR #8906
